src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -472,13 +480,13 @@ public void appendMultipleQueryStringsToUrl(String[] queryStrings) {
   public void pressDownArrow(WebElement element) {
     JavascriptExecutor js = (JavascriptExecutor) driver;
     js.executeScript("var e = jQuery.Event(\"keydown\"); "
-        + "e.which=40; $(arguments[0]).trigger(e);", element);
+                     + "e.which=40; $(arguments[0]).trigger(e);", element);
   }
 
   public void setDisplayStyle(String selector, String style) {
     JavascriptExecutor js = (JavascriptExecutor) driver;
     js.executeScript("document.querySelector(arguments[0]).style.display = arguments[1]", selector,
-        style);
+                     style);
   }
 
   private void purge(String url) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)